test/specs/carousel.test.js

```diff
@@ -363,6 +363,7 @@ describe('<Carousel />', () => {
       expect(wrapper).toHaveState('currentSlide', 2);
       wrapper.setProps({ children: [] });
       expect(wrapper).toHaveState('currentSlide', 0);
+      wrapper.update();
```

Comment:
This test suddenly started failing with the change unless I triggered a wrapper update to sync the latest render updates.